packages/contracts/test/contracts/L2/predeploys/OVM_GasPriceOracle.spec.ts

```diff
@@ -26,9 +26,9 @@ describe('OVM_GasPriceOracle', () => {
       await signer1.getAddress()
     )
 
-    OVM_GasPriceOracle.setOverhead(2750)
-    OVM_GasPriceOracle.setScalar(1500000)
-    OVM_GasPriceOracle.setDecimals(6)
+    await OVM_GasPriceOracle.setOverhead(2750)
+    await OVM_GasPriceOracle.setScalar(1500000)
+    await OVM_GasPriceOracle.setDecimals(6)
```

Comment:
how was this test suite working without this?

Reply:
Does it just become a race condition that always worked out for us?

Reply:
Yup exactly